### PR TITLE
Added example JSON config

### PR DESCRIPTION
**Pull request by pranay101**
hey @CillianFn, I created the first JSON file. it is facing some internal error which I really don't know about. hope you could help

### CODE REVIEW

File: configs/era5_example_config_local_run.json
Comment (alxmrs) [inline configs/era5_example_config_local_run.json:11]: I would expect these values to be a in a list.

File: configs/era5_example_config_local_run.json
Comment (alxmrs) [inline configs/era5_example_config_local_run.json:20]: I also expect these values to be in a list.

**Review by alxmrs — COMMENTED** [review]
Left a few notes. After making these adjustments, if there are still errors, I'll put the branch locally for testing.

**Comment by alxmrs** [comment]
Hey @pranay101, thanks for your contribution. We're happy to accept patches from you, but first, will you accept our contribution license agreement? See [this section](https://github.com/google/weather-tools/blob/main/CONTRIBUTING.md#contributor-license-agreement) of our contributing guide.

**Comment by pranay101** [comment]
hey @alxmrs, done with the desired changes.

**Comment by alxmrs** [comment]
Thanks! That's good to hear. I'm happy to merge these changes once #56 lands.

**Comment by pranay101** [comment]
@alxmrs do I need to create JSON for other example CGF files??

**Comment by alxmrs** [comment]
I think having one JSON example for both a MARS and Copernicus config is sufficient.

**Comment by alxmrs** [comment]
Hey @pranay101, wanted to follow up. Can you add one more example JSON file for the MARS config example? Also, please merge from main, and I want to make sure that this branch passes the new automated checks for configs.

**Comment by pranay101** [comment]
> 

@alxmrs yeah sure

**Comment by pranay101** [comment]
hey @alxmrs `raise APIException("ecmwf.API error 1: %s" % (self.last["error"],))
ecmwfapi.api.APIException: "ecmwf.API error 1: User 'anonymous' has no access to services/mars [while running 'FetchData']"`

what does this error mean, any ideas??

**Comment by alxmrs** [comment]
Yes, we discuss the cause of this error in #35. The MARS API requires a paid license.

**Comment by alxmrs** [comment]
Yes, we discuss the cause of this error in #35. The MARS API requires a paid license.

**Comment by pranay101** [comment]
> Yes, we discuss the cause of this error in #35. The MARS API requires a paid license.

Anyway i can test the json version of it. Tho it looks fine to me, pushing the code without testing wouldn't be so great 😅😅

**Comment by alxmrs** [comment]
Sorry for the lack of response. The MARS configs should work as is. If the JSON conversion parses, these should work as well. Would you like to add the config? If not, i'm happy merging this change as is.

**Comment by pranay101** [comment]
Hey @alxmrs, I don't think I should commit any code without assuring if it works or not. since I couldn't check the mars config I don't think we should merge it. Tho the config JSON is ready. Or If you have a paid license can you confirm that JSON config works if I attach the code here and then I can push the change. any way you want.

**Comment by alxmrs** [comment]
I'm happy to test it for you if you commit a JSON version of the MARS config. How does that sound?

**Comment by pranay101** [comment]
done sir

File: configs/mars_example_config.json
Comment (alxmrs) [inline configs/mars_example_config.json:20]: To my knowledge, this type of date range isn't supported for the JSON use case (yet).

File: configs/mars_example_config.json
Comment (alxmrs) [inline configs/mars_example_config.json:14]: Please use a standard JSON array instead of the `/` delimiter syntax. Same with the `step` below.

**Review by alxmrs — CHANGES_REQUESTED** [review]

File: configs/mars_example_config.json
Comment (pranay101) [reply configs/mars_example_config.json]: hey, @alxmrs, done with the desired changes replaced all delimiters with the array.

**Comment by pranay101** [comment]
hey, @alxmrs. Are any further changes in the code required?

**Comment by alxmrs** [comment]
> hey, @alxmrs. Are any further changes in the code required?

Hey @pranay101, I'm sorry I dropped the ball on this PR. I'm tested it now and it LGMT. As soon as this passes tests, I'll merge the change.

**Review by alxmrs — APPROVED** [review]

**Pull request merged**
